src/ApiException.php

```diff
@@ -137,18 +140,29 @@ public function getErrorInfoMetadata()
         return ($this->decodedMetadataErrorInfo) ? $this->decodedMetadataErrorInfo['errorInfoMetadata'] : null;
     }
 
+    /**
+     * Returns the unserialized errors
```

Comment:
(not familiar with the code base) What does "unserialized" errors mean?

Reply:
In this case, serialized would be binary form and unserialized would be in message form (if possible)